docs/freqai-outlier-detection.md

```diff
@@ -0,0 +1,79 @@
+# Outlier detection

```

Comment:
I'm not sure i understand why this deserves it's own page. 
It'd be better placed under "feature-engineering" if you ask me - right now that feature engineering part is kindof split in 2 - for no good reason, really - as this is also just part of feature engineering.

Reply:
I agree. They were on that page before so I'm moving them back.